.vscode/settings.json

```diff
@@ -24,7 +24,7 @@
   "python.formatting.provider": "black",
   "[python]": {
     "editor.codeActionsOnSave": {
-      "source.organizeImports": true
+      "source.organizeImports": "explicit"
```

Comment:
This can sometimes get accidentally committed when we don't want it committed - checking the files changed tab in your PR can help identify that you've committed it and then you can revert it